flags.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+)
+
+func parseFlags() []string {
+	var (
+		printHelp    bool
+		printVersion bool
+		cfgFile      string
+		version      = "HEAD"
+	)
+
+	flagSet := pflag.NewFlagSet("d8-lint", pflag.ContinueOnError)
+
+	flagSet.BoolVarP(&printHelp, "help", "h", false, "help message")
+	flagSet.BoolVarP(&printVersion, "version", "v", false, "version message")
+	flagSet.StringVarP(&cfgFile, "config", "c", "", "config file (default is $HOME/.d8lint.yaml)")
```

Comment:
`$HOME/.d8lint.yaml` should be `$(pwd)/.d8lint.yaml` or something like that

Reply:
Or event `target-dir/.d8lint.yaml`
Because, if I run `d8-lint ../some-dir-with-module`, I would expect config file to be read from ../some-dir-with-module/./d8lint.yaml`

`./` -> `./.d8-lint.yaml`
`.` -> `.d8-lint.yaml`
`/deckhouse` -> `/deckhouse/.d8-lint.yaml`

Reply:
We can pass multiple directories in the command line. Search all directories? And if not found, search the current directory and all above?

Reply:
Now we are looking for a configuration file from the current directory and all directories above it. Something like this:

```plain
time=2024-09-24T09:46:52.771+03:00 level=INFO msg="Dirs: [/deckhouse/modules/ /deckhouse/ee/modules/]"
time=2024-09-24T09:46:52.771+03:00 level=INFO msg="Config search paths: [./ /Users/devsyukov/src/github.com/deckhouse/d8-lint /Users/devsyukov/src/github.com/deckhouse /Users/devsyukov/src/github.com /Users/devsyukov/src /Users/devsyukov /Users /]"
```


In principle, you can try adding the transferred directories to the search list.

Reply:
Changed the processing, passing a list of directories to Loader. Now the configuration files are searched in these directories:

```plain
time=2024-09-24T10:06:58.589+03:00 level=INFO msg="Dirs: [/deckhouse/modules/ /deckhouse/ee/modules/]"
time=2024-09-24T10:06:58.589+03:00 level=INFO msg="Config search paths: [./ /deckhouse/modules /deckhouse / /Users/devsyukov]"
```